Libs/RaptureAPI/src/main/java/rapture/dsl/idef/DisplayLocator.java

```diff
@@ -42,6 +42,10 @@ public String toString() {
 	@Override
 	public Object value(String key, Map<String, Object> mappedContent,
 			DocumentMetadata meta) {
-		return key.split("/")[fieldNum];
+		try {
+            return key.split("/")[fieldNum];
+        } catch (ArrayIndexOutOfBoundsException e) {
```

Comment:
would it be better to check here instead of relying on an exception?

also, the formatting is off